src/Propel/Common/Config/Loader/IniFileLoader.php

```diff
@@ -151,6 +152,7 @@ private function parseKey(string $key, $rawValue, array &$config): void
                 $value = (float)$rawValue;
             }
         }
+        // @phpstan-ignore-next-line
```

Comment:
Lets remove all of those, we never need to use them.

Reply:
There are a number of cases where `static` is being used to access a private or protected method.  I'm assuming it's safe to assume `self` is sufficient?

Reply:
Or we change to protected, depending on the case.